dev/dll/Microsoft.UI.Xaml.vcxproj.filters

```diff
@@ -108,5 +108,25 @@
     <Page Include="$(OutDir)Generic.xaml" />
     <Page Include="$(OutDir)rs1_themeresources.xaml" />
     <Page Include="@(PageRequiringCustomCompilation)" />
+    <Page Include="@(PageRequiringCustomCompilation)" />
```

Comment:
Is this duplication intentional?

Reply:
No. There is just something weird going on with Visual Studio that causes it to add these dupes. I'll remove them.